src/store.tsx

```diff
@@ -58,7 +59,21 @@ const PokemonContext = createContext({
   deck$,
 });
 
-export const usePokemon = () => useContext(PokemonContext);
+export const usePokemon = () => {
+  const { pokemon$ } = useContext(PokemonContext);
+  return [useObservableState(pokemon$, [])];
```

Comment:
I don't think usePokemon or useDeck need to return single item arrays. They could just return the observable.

Reply:
Yes, you can always have access to the observable via useContext.